### PR TITLE
More R2/Radare -> Rizin renames

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [x] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [x] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

Apparently some "radare" and "r2" mentions were still in the sources, comments, documentation, and various supplementary files. Renamed all of them except r2dec/r2dec-js since I don't know what to do with it (cc @wargio )

Also I think some of these files are unnecessary or should be updated regarding the Rizin building process. But this is out of the scope of the renaming PR.

**Test plan (required)**

Everything green

### CODE REVIEW

File: src/Cutter.pro [hunk 1]
```diff
@@ -278,23 +278,23 @@ QMAKE_CXXFLAGS += -g
...
-macx:CUTTER_BUNDLE_R2_APPBUNDLE {
-    message("Using r2 rom AppBundle")
+macx:CUTTER_BUNDLE_RZ_APPBUNDLE {
+    message("Using rizin rom AppBundle")
     DEFINES += MACOS_RZ_BUNDLED
 }
 
 CUTTER_APPVEYOR_R2DEC {
-    message("Appveyor r2dec")
+    message("Appveyor rdec")
```
Comment (ITAYC0HEN) [inline src/Cutter.pro:287]: Isn't it "jsdec" or something like this?
@wargio

**Review by ITAYC0HEN — COMMENTED** [review]
Looks good to me, compiled successfully. But need to carefully check the extended implications on CI and non-Linux machines to verify that it is not breaking zsomething

File: src/Cutter.pro [hunk 2]
```diff
@@ -383,8 +383,8 @@ SOURCES += \
     dialogs/MapFileDialog.cpp \
     common/CommandTask.cpp \
     common/ProgressIndicator.cpp \
-    common/R2Task.cpp \
-    dialogs/R2TaskDialog.cpp \
+    common/RzTask.cpp \
+    dialogs/RzTaskDialog.cpp \
```
Comment (thestr4ng3r) [inline src/Cutter.pro:387]: I would rename these to `RizinTask...` instead. I have some local changes that do this here so maybe better remove the actual code renames (except strings and stuff) from this pr.

**Review by thestr4ng3r — CHANGES_REQUESTED** [review]

**Comment by XVilka** [comment]
You can just push on top of this PR too.

**Comment by thestr4ng3r** [comment]
Yeah let's first merge #2517, then update this pr on top.

**Comment by XVilka** [comment]
The only missing rename is "r2dec" and "R2DEC".

**Review by thestr4ng3r — APPROVED** [review]
Looks perfect now.

**Pull request merged**